packages/functional-tests/tests/syncV3/signIn.spec.ts

```diff
@@ -20,129 +21,177 @@ test.describe('severity-2 #smoke', () => {
       test.slow();
     });
 
-    test('verified, does not need to confirm', async ({
+    test('verified email, does not need to confirm', async ({
       target,
       credentials,
       syncBrowserPages,
     }) => {
       const email = credentials.email;
-      const { page, login, connectAnotherDevice, signinTokenCode } =
-        syncBrowserPages;
+      const { page, login, connectAnotherDevice } = syncBrowserPages;
+
       await page.goto(
         `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email`
       );
-      await login.setEmail(email);
-      await signinTokenCode.clickSubmitButton();
-      await login.setPassword(PASSWORD);
-      await login.submit();
+      await login.login(email, PASSWORD);
+
       await expect(connectAnotherDevice.fxaConnected).toBeEnabled();
     });
 
-    test('verified, resend', async ({ emails, target, syncBrowserPages }) => {
-      const { configPage, page, login, connectAnotherDevice, signinTokenCode } =
+    test('verified email, can ask to resend code', async ({
+      emails,
+      target,
+      syncBrowserPages,
+    }) => {
+      const { page, login, connectAnotherDevice, signinTokenCode } =
         syncBrowserPages;
+
       const [email] = emails;
-      const config = await configPage.getConfig();
-      test.fixme(
-        config.showReactApp.signUpRoutes,
-        'this test goes through the signup flow instead of sign in, skipping for react, see FXA-8973'
-      );
+
+      // Simulate a new sign up that requires a signin verification code.
+      target.auth.signUp(email, PASSWORD, {
+        service: 'sync',
+        preVerified: 'true',
+      });
 
       await page.goto(
         `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email`
       );
-      await login.setEmail(email);
-      await signinTokenCode.clickSubmitButton();
```

Comment:
This is a good example of something that was a false positive. This just so happened to work because the `signinTokenCode` submit button had the same selector as the login submit button. Once the `checkForPath()` was introduced operation failed, as it should. A code is not being submitted on the `signInTokenCode` page, and so having this action work is actually misleading.